install_test/CMakeLists.txt

```diff
@@ -5,13 +5,13 @@
 cmake_minimum_required(VERSION 3.22)
 project(test-installed-ddc LANGUAGES CXX)
 
-find_package(DDC REQUIRED)
+find_package(DDC REQUIRED COMPONENTS fft pdi splines)
 
 message("DDC options:")
 message("DDC_BUILD_DOUBLE_PRECISION=${DDC_BUILD_DOUBLE_PRECISION}")
-message("DDC_BUILD_KERNELS_FFT=${DDC_BUILD_KERNELS_FFT}")
-message("DDC_BUILD_KERNELS_SPLINES=${DDC_BUILD_KERNELS_SPLINES}")
-message("DDC_BUILD_PDI_WRAPPER=${DDC_BUILD_PDI_WRAPPER}")
+message("DDC_fft_FOUND=${DDC_fft_FOUND}")
+message("DDC_splines_FOUND=${DDC_splines_FOUND}")
```

Comment:
Maybe a small cosmetic improvement if you put this line below in order to keep the order "fft pdi splines" that you seem to use elsewhere.

Reply:
True I tried to follow this rule, i will modify